lib/net/pktdrv/pktdrv.h

```diff
@@ -1,6 +1,10 @@
 #ifndef _Pktdrv_
 #define _Pktdrv_
 
+#ifdef __cplusplus
+extern "C" {
+#endif

```

Comment:
I wonder if we should even C++-ify all of our headers.
If it's well defined that certain APIs are C APIs, then I don't see the issue with doing it like this instead:

```c
// (in app.cpp)

#include <windows.h>

extern "C" {
#include "foo.h"
}
```

Reply:
I think we should - people usually expect C headers to just work in C++, even though the languages are quite different internally, and I consider it to be much cleaner to adjust the header once.